AmplifyPlugins/DataStore/DataStoreCategoryPlugin.xcodeproj/xcshareddata/xcschemes/AWSDataStoreIntegrationV2Scenario3V2Tests.xcscheme

```diff
@@ -100,6 +100,9 @@
                <Test
                   Identifier = "DataStoreScalarTests">
                </Test>
+               <Test
+                  Identifier = "DataStoreSchemaDriftTests">
```

Comment:
Maybe we can have a section like < SelectedTests > which will only include the tests corresponding to one scheme, which can help making number of files less in a PR
https://stackoverflow.com/a/63879084

Reply:
I'll capture this as an issue to explore further to improve the set up of our tests. Currently it's a hassle when adding a new test, we have to disable it from all other schemas except for the schema we want it in (be that a new or existing schema). That link you posted describes using Xcode Test Plans, so I think we should explore that and extend it to other plugins as well if it makes sense.